lib/mongoid/railties/controller_runtime.rb

```diff
@@ -43,7 +43,7 @@ module ClassMethods
           def log_process_action(payload)
             messages = super
             mongoid_runtime = payload[:mongoid_runtime]
-            messages << ("MongoDB: %.1fms" % mongoid_runtime.to_f) if mongoid_runtime
+            messages << ("MongoDB: %.1fms" % (mongoid_runtime * 1000.0)) if mongoid_runtime
```

Comment:
Maybe it would be better to multiply by 1000 on this line:

```
          self.mongoid_runtime = mongo_rt_before_render + mongo_rt_after_render
          runtime - mongo_rt_after_render
```

That way the variable `mongoid_runtime` in the payload would be in milliseconds.

We should take a peek with what ActiveRecord does for this and try to be consistent.